django/db/migrations/loader.py

```diff
@@ -64,6 +64,10 @@ def load_disk(self):
         self.unmigrated_apps = set()
         self.migrated_apps = set()
         for app_config in apps.get_app_configs():
+            if (self.connection is not None and
```

Comment:
Agreed, but migrations_module() is a classmethod that acts like a staticmethod so it doesn't have access to self.connection. I couldn't came up with a backward compatible solution. Do you have a suggestion?